near-sdk/src/lib.rs

```diff
@@ -102,26 +102,172 @@ extern crate quickcheck;
 /// to generate the necessary code to expose `pub` methods from the contract as well
 /// as generating the glue code to be a valid NEAR contract.
 ///
-/// The macro is a syntactic sugar for [macro@near_bindgen] and expands to the [macro@near_bindgen] macro invocations.
+/// The macro is a syntactic sugar for [near_bindgen] and expands to the [near_bindgen] macro invocations.
 ///
```

Comment:
this should link to `#[near(serializers=[...])]` (fuller syntax to avoid confusion) and `contract_state` headings, 
noting that `near_bindgen` doesn't support these and supports all the rest of sub-attributes